keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -293,33 +291,49 @@ def _augment(self, inputs):
         return result
 
     def _batch_augment(self, inputs):
-        return self._map_fn(self._augment, inputs)
+        augmented_batch = self._map_fn(self._augment, inputs)
+        return augmented_batch
 
     def _format_inputs(self, inputs):
         if tf.is_tensor(inputs):
             # single image input tensor
-            return {IMAGES: inputs}, False, False
+            meta_data = {"is_dict": False, "use_targets": False}
+            inputs = {IMAGES: inputs}
```

Comment:
lets define metadata above the if statements.

```
if not isinstance(inputs, dict):
    raise ValueError(
                f"Expect the inputs to be image tensor or dict. Got {inputs}"
     )
metadata = {"is_dict": True"}

if TARGETS in inputs:
     ....
     metadata["use_targets"] = True

if BOUNDING_BOXES in inputs:
    ...

return inputs metadata

Reply:
done

Reply:
done!